src/datepicker/datepicker-navigation.ts

```diff
@@ -0,0 +1,56 @@
+import {Component, Input, Output, EventEmitter} from '@angular/core';
+import {NavigationEvent} from './datepicker-view-model';
+import {NgbDate} from './ngb-date';
+import {NgbDatepickerI18n} from './datepicker-i18n';
+import {NgbCalendar} from './ngb-calendar';
+
+@Component({
+  selector: '[ngbDatepickerNavigation]',
+  styles: [`
+    td {
+      text-align: center;
+      padding-bottom: 0.25rem;
+    }
+  `],
+  template: `
+    <tr>
```

Comment:
This restricts this component to being inside a `tbody` element - is this what we want?